node_tree.py

```diff
@@ -307,6 +478,46 @@ def draw_color(self, context, node):
         return self.nodule_color
 
 
+class SvLinkMatrix(bpy.types.Operator):
```

Comment:
Fair enough. We’ll deal with overlap later. Btw. the “small usability details” help me understand more about blender/sverchok node coding... i don’t just do it for the detail sake.